File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/WeatherResponse.java
package com.nerdery.umbrella.data.model;

import com.google.gson.annotations.SerializedName;

public class WeatherResponse {
    @SerializedName("timezone")
    String timeZone;

    @SerializedName("currently")
    ForecastCondition currentForecast;

    HourlyResponse hourly;

    public String getTimeZone() {
        return timeZone;
    }

    public ForecastCondition getCurrentForecast() {
        return currentForecast;
    }
}

File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/WeatherResponse.kt
package com.nerdery.umbrella.data.model

import com.google.gson.annotations.SerializedName

/**
 * Response from DarkSky weather requests in [com.nerdery.umbrella.data.api.WeatherService]
 */
data class WeatherResponse (

    /**
     * Current Weather Condition
     * @return ForecastCondition
     */
    @SerializedName("currently")
    var currentForecast: ForecastCondition?,
    /**
     * Hourly Response model that contains list of ForecastConditions
     * @return HourlyResponse
     */
    var hourly: HourlyResponse?
)
